examples/user_courses.py

```diff
@@ -0,0 +1,72 @@
+
+import json
+import requests
+
+client_id = "..."
+client_secret = "..."
+
+class StepicAPI(object):
+    def __init__(self, client_id, client_secret):
+        self.api_url = 'https://stepic.org/api/'
+        
+        self.client_id = client_id
+        self.client_secret = client_secret
+        
+        try:
+            auth = requests.auth.HTTPBasicAuth(client_id, client_secret)
+            resp = requests.post('https://stepic.org/oauth2/token/',
+                         data={'grant_type': 'client_credentials'},
+                         auth=auth
+                         )
+            self.token = json.loads(resp.text)['access_token']
+        except:
+            print("Error while obtaining token")
+      
+    def get(self, url):
+        try:
+            resp = json.loads(requests.get(url, headers={'Authorization': 'Bearer ' + self.token}).text)
+        except:
+            print("Error while getting data")
+            resp = None
+        return resp
+     
+    # announcements
+    def announcements(self, page = 1):
+        url = self.api_url + 'announcements?page=%d'%page 
```

Comment:
It's preferable to use a new format string syntax: https://docs.python.org/3.4/library/string.html#format-examples